lib/cdo.rb

```diff
@@ -148,9 +153,14 @@ def javabuilder_url(path = '', scheme = '')
 
     def javabuilder_upload_url(path = '', scheme = '')
       if rack_env?(:development)
-        # To use a developer instance of Javabuilder instead, replace this url with
-        # 'https://<your-javabuilder-domain>-http.dev-code.org/seedsources/sources.json'
-        'http://localhost:8080/javabuilderfiles/seedsources'
+        # On localhost, we default to using the "test" Javabuilder stack. To point
+        # to your Javabuilder WebSocket server running on localhost, set
+        # 'use_localhost_javabuilder: true' in your locals.yml. To point to a
+        # deployed development instance of Javabuilder, set
+        # 'local_javabuilder_stack_name: "your stack name"' in your locals.yml.
+        return 'http://localhost:8080/javabuilderfiles/seedsources' if CDO.use_localhost_javabuilder
+        stack_name = CDO.local_javabuilder_stack_name || 'javabuilder-test'
+        "https://#{stack_name}-http.code.org/seedsources/sources.json"
```

Comment:
we may have to split this out if we end up doing adhocs on the dev environment, but I think we are sticking with prod for now.

Reply:
Ah yep good call. We can update this logic once we switch over